rust/nft-candy-machine/src/lib.rs

```diff
@@ -76,6 +77,38 @@ pub mod nft_candy_machine {
                 token_program: ctx.accounts.token_program.to_account_info(),
                 amount: candy_machine.data.price,
             })?;
+
+            if let Some(secondary_mint) = candy_machine.secondary_token_mint {
+                let secondary_wallet = &ctx.remaining_accounts[2];
+                let secondary_token_account_info = &ctx.remaining_accounts[3];
+                let transfer_authority_info = &ctx.remaining_accounts[1];
+                let secondary_token_account: spl_token::state::Account =
+                    assert_initialized(&secondary_token_account_info)?;
+
+                assert_owned_by(&secondary_token_account_info, &spl_token::id())?;
+                // Need to check that this destination wallet is owned by the program?
+                // assert_owned_by(&secondary_wallet, &spl_token::id())?;
```

Comment:
i am unsure of what to do here — i assume anchor is not validating these the way they do with the accounts in `context`. we're using `remaining_accounts` because this is an optional account to pass, so i _think_ i need to check that the parameterized `secondary_wallet` is... the wallet that we own?